blockchain/blockchain-wallet-v4-frontend
# Removed text and functionality related to "What's New" flyover

## Description (optional)
Removes the "What's New" link from the gear settings dropdown, as well as removes the associated flyover, and other code related to what's new functionality.

## Testing Steps (optional)
- Login, click the gear in the upper right corner, see that "What's New" is now gone.
- Make sure the rest of the application is working normally.

*Note to QA
- `notificationsLink` e2d tag has been removed.
